bugswarm/client/docker.py

```diff
@@ -87,7 +87,14 @@ def docker_pull(image_tag):
     command = 'sudo docker pull {}'.format(image_location)
     _, _, returncode = ShellWrapper.run_commands(command, shell=True)
     if returncode != 0:
-        log.error('Could not download the image', image_location, 'from Docker Hub.')
+        # We try to pull from bugswarm/images instead of cached-images
```

Comment:
Let's make the comment: `# Image is not cached. Attempt to pull from bugswarm/images`.